pkg/rpc/adminservice/base.go

```diff
@@ -50,6 +51,8 @@ func (m *AdminService) interceptPanic(ctx context.Context, request proto.Message
 
 const defaultRetries = 3
 
+var workflowExecutorReconnectDelay = 30 * time.Second
```

Comment:
If we are making the number of reconnect attempts configurable, should we make this too?

Reply:
sure